src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+                } else if (firstWord.equals("todo")) {
+                    if (arr.length > 1) { //check for errors
+                        //add task to list
+                        ToDos t = new ToDos(arr[1]);
+                        list.add(t);
+                        //print output
+                        System.out.println("Got it. I've added this task:");
+                        System.out.println(t.toString());
+                        System.out.println("Now you have " + list.size() + " tasks in the list.");
+                    } else {
+                        throw new DukeException("☹ OOPS!!! The description of a todo cannot be empty.");
+                    }
+                } else if (firstWord.equals("deadline")) {
+                    if (arr.length > 1) { //check for errors
+                        //separate task and deadline
+                        String arr2[] = arr[1].split("/", 2);
+
+                        if (arr2.length > 1) { //check that deadline is specified
+                            String arr3[] = arr2[1].split(" ", 2);
+                            //add task and deadline to list
+                            Deadline d = new Deadline(arr2[0], arr3[1]);
+                            list.add(d);
+                            //print output
+                            System.out.println("Got it. I've added this task:");
+                            System.out.println(d.toString());
+                            System.out.println("Now you have " + list.size() + " tasks in the list.");
+                        } else {
+                            throw new DukeException("☹ OOPS!!! Please specify the deadline.");
+                        }
+                    } else {
+                        throw new DukeException("☹ OOPS!!! The description of a deadline cannot be empty.");
+                    }
+                } else if (firstWord.equals("event")) {
+                    if (arr.length > 1) { //check for errors
+                        //separate task and time
+                        String arr2[] = arr[1].split("/", 2);
+
+                        if (arr2.length > 1) {
+                            String arr3[] = arr2[1].split(" ", 2);
+                            //add task and deadline to list
+                            Events e = new Events(arr2[0], arr3[1]);
+                            list.add(e);
+                            //print output
+                            System.out.println("Got it. I've added this task:");
+                            System.out.println(e.toString());
+                            System.out.println("Now you have " + list.size() + " tasks in the list");
+                        } else {
+                            throw new DukeException("☹ OOPS!!! Please specify event time.");
+                        }
+                    } else {
+                        throw new DukeException("☹ OOPS!!! The description of an event cannot be empty.");
```

Comment:
Could consider refactoring each if-else case as methods to avoid nested if-else blocks to improve readability.